features/steps/payment_methods_steps.py

```diff
@@ -598,13 +598,14 @@ def step_impl(context, e2e_config : e2eConfig, jwt_config : JwtConfig):
 @step("User opens (?:example page|example page (?P<example_page>.+))")
 def step_impl(context, example_page : ExamplePage):
     payment_page = context.page_factory.get_page(page_name='payment_methods')
+    jwt = str(encode_jwt_for_json(JwtConfig.BASE_JWT), "utf-8")
```

Comment:
For updateJwt it shouldn't be JwtConfig.BASE_JWT (it's the same as original jwt). You can use some other value with different baseAmount, currency, etc.